### ISSUE TITLE
Upgrade to version 4.5.0 causes AttributeError: 'Chrome' object has no attribute 'backend'

### ISSUE THREAD

**Issue by jglorioso** [wkeeling/selenium-wire#374]
Using undetected chromedriver and launching as so:
```
import seleniumwire.undetected_chromedriver.v2 as uc

driver = uc.Chrome(
    executable_path=UndetectedChrome.chromedriver_executable_path,  # resolves to a local patched exe
    seleniumwire_options=proxy_options,  # This was None
    options=chrome_options,  # chrome_options.to_capabilities() pasted below
    delay=0)
```

The chrome options were:
```
{'browserName': 'chrome', 'version': '', 'platform': 'ANY', 'pageLoadStrategy': 'normal', 'goog:chromeOptions': {'extensions': [], 'args': ['--user-data-dir=lib/chrome_profile', '--no-first-run', '--password-store=basic', '--no-service-autorun', '--disable-blink-features=AutomationControlled']}}
```

Stack:
```
Traceback (most recent call last):
  File "dev/launch_selenium.py", line 43, in <module>
    driver_options=driver_options)
  File "<redacted>/undetected_chrome.py", line 75, in get_driver
    delay=0)
  File "<redacted>/.venv/dev/lib/python3.7/site-packages/seleniumwire/undetected_chromedriver/v2.py", line 27, in __init__
    self.proxy = backend.create(port=seleniumwire_options.get('port', 0), options=seleniumwire_options)
  File "<redacted>/.venv/dev/lib/python3.7/site-packages/seleniumwire/webdriver.py", line 96, in proxy
    self.backend.master.options.update(**build_proxy_args(get_upstream_proxy({'proxy': proxy_conf})))
AttributeError: 'Chrome' object has no attribute 'backend'
Exception ignored in: <function Chrome.__del__ at 0x10a47e710>
Traceback (most recent call last):
  File "<redacted>/.venv/dev/lib/python3.7/site-packages/undetected_chromedriver/v2.py", line 623, in __del__
    self.quit()
  File "<redacted>/.venv/dev/lib/python3.7/site-packages/seleniumwire/webdriver.py", line 49, in quit
    self.backend.shutdown()
AttributeError: 'Chrome' object has no attribute 'backend'
```

**Comment by wkeeling** [comment]
Many thanks for flagging this. Now fixed in v4.5.1.

**Comment by jglorioso** [comment]
Thanks for the quick turnaround, I just happened to be rebuilding my project right after the 4.5 release! Can confirm 4.5.1 resolves the issue.
